frontend/discourse/select-kit/components/select-kit.js

```diff
@@ -164,21 +165,21 @@ export default class SelectKit extends Component {
   @protoProp content = null;
   @protoProp value = null;
   @protoProp selectKit = null;
-  @protoProp mainCollection = null;
-  @protoProp errorsCollection = null;
   @protoProp options = null;
   @protoProp valueProperty = "id";
   @protoProp nameProperty = "name";
   @protoProp labelProperty = null;
   @protoProp titleProperty = null;
   @protoProp langProperty = null;
+  @trackedArray mainCollection = null;
+  @trackedArray errorsCollection = null;
```

Comment:
Looks like nothing is overriding those two props so it's safe for them to not be `@protoProp` 👍